### DISCUSSION TITLE
What's going on with the website and forums?

### DISCUSSION THREAD

**Discussion by MrConorAE** [Pryaxis/TShock#2335] [Q&A] [answered]
Hi there, TShock community!

I'm trying to install WorldEdit for TShock (as a plugin), but the forums seem to have broken. Taking a look at the dev console shows a completely empty document and nothing in the console. Also, the website link listed in GitHub (tshock.co) just redirects to this GitHub repo.

I understand that the forums were intended to be phased out, but if that's the case, where do I go to get things like WorldEdit now?

Thanks,
MrConorAE

**Comment by Kojirremer** [comment, marked as answer]
There is a plugin subrepo listed on TShock's project page. Unfortunately, some of the plugins there are already outdated, but if you know how to search for Github forks you can find updated builds here and there.

EDIT: Updated WorldEdit, BTW: https://github.com/Stealownz/WorldEdit/releases

**Reply by MrConorAE** [reply]
Thanks!

**Comment by hakusaro** [comment]
Recently, I rebuilt the forums on a Digital Ocean Clear Linux OS image, but unfortunately some unknown error has apparently occurred. Unfortunately, I can't currently reboot the instance. However, I'll be working to restore service shortly. Generally, @Kojirremer is correct, in the senes that plugins on the old forums likely do not work anymore.

**Comment by hakusaro** [comment]
The forums are back up!

**Reply by MrConorAE** [reply]
Perfect, thank you so much!
